gradle.properties

```diff
@@ -11,37 +11,36 @@
 dockerExecutable=/usr/bin/docker
 
 #3rd party Versions
-apacheCuratorVersion=2.11.0
-checkstyleToolVersion=7.1
-findbugsVersion=3.0.1
+apacheCommonsCsvVersion=1.5
+apacheCuratorVersion=4.0.+
```

Comment:
Apparently the use of `+` for version format causes issues when publishing to maven central, see it in PR #1929. Let's use the full version instead.

Reply:
There are a bunch of uses of `+` here, please fix all those.